llama_stack/apis/safety/safety.py

```diff
@@ -43,10 +45,12 @@ async def get_shield(self, identifier: str) -> Shield: ...
 
 
 @runtime_checkable
+@trace_protocol
```

Comment:
consider using a TracedProtocol base class.

Reply:
yes that makes sense